types/resize-observer-browser/index.d.ts

```diff
@@ -4,14 +4,20 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.7
 
+declare global {
+    interface Window {
```

Comment:
You want to add `YourLibraryName-global.test.ts` in the tests as well to verify the globals are working as intended
Refer https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

Reply:
Done, added into a single commit.